conductor_api/src/conductor/base.rs

```diff
@@ -750,7 +750,11 @@ impl Conductor {
 
                 // Get DNA
 
-                let mut dna_config = self.config.dna_by_id(&instance_config.dna).unwrap();
```

Comment:
Was this a latent bug before? `dna_by_id` clones its result so I doubt that mutating it has the desired effect. If so, it might be good to add a regression test.